pilot/info/jobdata.py

```diff
@@ -57,6 +58,7 @@ class JobData(BaseData):
     platform = ""   # cmtconfig value from the task definition
 
     is_eventservice = False        # True for event service jobs
+    is_eventservicemerge = False   # True for event service merge jobs
```

Comment:
The version of this file is not the same as the one I committed yesterday. E.g. the 'status' field should now be a dictionary. Looking at the build number, you have build 19, but it should be 80..

Reply:
rebased with your code.

Reply:
I rebased with your code. Now status is dict.